woocommerce/woocommerce-ios
# [Mobile Payments] Remove unreferenced fancy alert

Closes #4260 

To test:
- Ensure the application still compiles
- No unit tests appear to have been affected

Update release notes:
